vuejs/index.html

```diff
@@ -0,0 +1,18 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <link rel="icon" type="image/svg+xml" href="/vite.svg" />
```

Comment:
I reckon this can be removed as the svg is no longer part of the source tree.

Reply:
done